### ISSUE TITLE
[$250] [Rules] Connect Rules to upgrade flows

### ISSUE THREAD

**Issue by garrettmknight** [Expensify/App#43915]
Rules have been released to beta in NewDot, we'll need to connect our upgrade flow in the Rules section of the workspace editor.

UI in the Upgrades doc [here](https://docs.google.com/document/d/1Pfy1xCt-RklqxYn1m37tGPUJNrYeNFh6UEpNsojaKZ4/edit#bookmark=id.i8266ad0gywy).

Translations for the upgrade copy are [here](https://docs.google.com/document/d/1nSWknL5X-Rh_yX0caTSbQQJ4mT8N5D3dzZC9WQg0O9o/edit#bookmark=id.7qfi8tyfqlzj).

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~012600ef2e91fde48d</li>
        <li>Upwork Job ID: 1821258868868350174</li>
        <li>Last Price Increase: 2024-08-07</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @allroundexperts</details>

**Comment by garrettmknight** [comment]
Still on hold pending BE issues

**Comment by garrettmknight** [comment]
Still on hold pending BE issues

**Comment by garrettmknight** [comment]
Still on hold pending implementation - this one actually might take a while so I could see us closing and just letting them update when it's done. Gonna leave it open until we have a confirmation on an internal convo though.

**Comment by garrettmknight** [comment]
Same as above, when we're ready to close out Upgrades, I'll just move this over.

**Comment by garrettmknight** [comment]
Same as above

**Comment by JmillsExpensify** [comment]
FYI, this can start being worked on. The relevant PRs will be on staging in the next deploy.

**Comment by garrettmknight** [comment]
Using external to get a C+ to review. @allroundexperts assigning you to complete since this is the last upgrade feature mentioned in the upgrades doc.

**Comment by garrettmknight** [comment]
@allroundexperts this was added to the doc later so I'm going to pay this out as a normal issue for you.

**Comment by allroundexperts** [comment]
I'm just waiting on https://github.com/Expensify/App/pull/46617 to get merged before adding this in.

**Comment by allroundexperts** [comment]
I see that https://github.com/Expensify/App/pull/47468 is still in draft state.

**Comment by garrettmknight** [comment]
Bumped that other issue/PR to see when they expect to complete.

**Comment by garrettmknight** [comment]
More changes requested - Going to put this one on Hold until https://github.com/Expensify/App/pull/47468 is complete.

**Comment by garrettmknight** [comment]
Still waiting on https://github.com/Expensify/App/pull/47468

**Comment by garrettmknight** [comment]
Still waiting on https://github.com/Expensify/App/pull/47468

**Comment by garrettmknight** [comment]
Still waiting on https://github.com/Expensify/App/pull/47468

**Comment by garrettmknight** [comment]
@allroundexperts good to go now that we're on staging?

**Comment by allroundexperts** [comment]
Checking now.

**Comment by allroundexperts** [comment]
This seems to have been added already [here](https://github.com/Expensify/App/pull/47083).

**Comment by garrettmknight** [comment]
Huh, looks like they went ahead and did it - @allroundexperts did you have a PR up?

**Comment by allroundexperts** [comment]
Nops, I did not.

**Comment by garrettmknight** [comment]
Cool, we'll just close this one then.
